docs/diagrams/RunningLowSequenceDiagram.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+!include style.puml
+
+participant ":RunningLowCommand" as RunningLowCommand
+participant "foodCategory :FoodCategory" as FoodCategory
+participant "fridge:Fridge" as Fridge
+participant "food :Food" as Food
+
+-> RunningLowCommand: execute()
+activate RunningLowCommand
+
+loop foodCategory in FoodCategory
+    RunningLowCommand -> FoodCategory: updateMessage()
+    activate FoodCategory
+    FoodCategory -> FoodCategory: getMinimumQuantity()
+    opt totalQuantity < minimumQuantity
```

Comment:
I think this might be incorrect? Where did "totalQuantity" come from? I believe the Fridge needs to read the value from somewhere. Should there be a clarification?